getting-started/templates/systemlink-values.yaml

```diff
@@ -343,6 +343,77 @@ testmonitorservice:
       ##
       certificateSubPath: *postgresCertificateFileName
 
+## Configuration for the Alarm Service
+##
+alarmservice:
+  ## Per-replica rate limiting configuration
+  rateLimits:
```

Comment:
We expose rate limiting config like this for other services, so I figured we'd do it for alarm too.